Control/Parallel/Strategies.hs

```diff
@@ -432,19 +432,33 @@ rpar  x = case (par# x) of { _ -> Done x }
 #endif
 {-# INLINE rpar  #-}
 
--- | instead of saying @rpar `dot` strat@, you can say
--- @rparWith strat@.  Compared to 'rpar', 'rparWith'
+-- | Perform a computation in parallel using a strategy.
 --
---  * does not exit the `Eval` monad
---
---  * does not have a built-in `rseq`, so for example `rparWith r0`
---    behaves as you might expect (it is a strategy that creates a
---    spark that does no evaluation).
+-- @
+-- rparWith strat x
+-- @
 --
+-- will spark a thread to perform @strat x@ in parallel. Note
```

Comment:
let's not use the terminology "thread" here to avoid confusion. How about "will spark @strat x@".